test/end-to-end/screenshots.yml

```diff
@@ -4,10 +4,10 @@ python_selenium_container: endtoend_python-selenium_1
 screenshot:
   names:
     # - patient_view_lgg_ucsf_2014_case_id_P04
-    - study_view_lgg_ucsf_2014
+    # - study_view_lgg_ucsf_2014
```

Comment:
@zhx828 seems bad not having a single e2e test in backend repo - maybe good to have one functioning one?

Reply:
I will add the test before merging to master